src/coreclr/scripts/superpmi_diffs.py

```diff
@@ -53,9 +63,10 @@ def setup_args(args):
                         "Invalid type \"{}\"".format)
 
     coreclr_args.verify(args,
```

Comment:
don't we need argument parsing for the newly renamed `host_os`?

Reply:
`host_os` is a little magic, as it's parsed by the `coreclr_arguments.py` script we use.